File: C/안항교육 반복문 풀이/03/03.c
#include<stdio.h>

int main() {
	int input;
	
	printf("정수를 입력하시오 >> ");
	scanf("%d", &input);

	printf("약수 : ");

	for (int i = 1; i <= input; i++) {
		if (input % i == 0)
			printf("%d ", i);
	}
}

File: PYTHON/1학년 2학기 교양/시험대비문제/03.py
# 3. 두 수를 입력 받아서 두 수가 모두 짝수이면 “모두 짝수”를
#    출력하고, 모두 홀수이면 “모두 홀수”를 출력하는 프로그램

a=int(input("input : "))
b=int(input("input : "))

if a%2==0 and b%2==0:
    print("모두 짝수")
if a%2==1 and b%2==1:
    print("모두 홀수")
    
